proto/pocket/migration/tx.proto

```diff
@@ -264,21 +245,53 @@ message MsgClaimMorseSupplier {
 // - The session end height in which the claim was committed
 // - The staked supplier
 message MsgClaimMorseSupplierResponse {
+
   // The hex-encoded address of the Morse account whose balance was claimed.
   // E.g.: 00f9900606fa3d5c9179fc0c8513078a53a2073e
   string morse_src_address = 1 [(gogoproto.jsontag) = "morse_src_address"];
-
+  
   // The unstaked balance which was claimed.
-  cosmos.base.v1beta1.Coin claimed_balance = 2 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "claimed_balance"];
-
+  cosmos.base.v1beta1.Coin claimed_balance        = 2 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "claimed_balance"       ];
   cosmos.base.v1beta1.Coin claimed_supplier_stake = 3 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "claimed_supplier_stake"];
-
+  
   // The session end height (on Shannon) in which the claim was committed (i.e. claimed).
   int64 session_end_height = 4 [(gogoproto.jsontag) = "session_end_height"];
-
+  
   // The supplier which was staked as a result of the claim.
   shared.Supplier supplier = 5 [(gogoproto.jsontag) = "supplier"];
 
   // Next free index: 6
 }
 
+message MsgRecoverMorseAccount {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // The bech32-encoded address of the migration module authority account ("gov" module address by default).
+  // ONLY the authority, or its delegates, MAY recover Morse recoverable accounts.
+  string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.jsontag) = "authority"];
+
+  // The bech32-encoded address of the Shannon account to which the Morse account's stake(s) and/or
+  // balance(s) will be minted (recovered).
+  string shannon_dest_address = 2 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.jsontag) = "shannon_dest_address"];
+
+  // The hex-encoded address of the recoverable Morse account whose stake(s) and/or balance(s) will be recovered.
+  // E.g.: 00f9900606fa3d5c9179fc0c8513078a53a2073e
+  string morse_src_address = 3 [(gogoproto.jsontag) = "morse_src_address"];
+
+  // Next free index: 4
```

Comment:
NIT: This usually goes at the top (below message).

Just following Google's best practices. I was an "official code reviewer" by many metrics.